src/components/Profile/Details.tsx

```diff
@@ -1,5 +1,6 @@
 import Follow from '@components/Shared/Follow';
 import Markup from '@components/Shared/Markup';
+import Message from '@components/Shared/Message';
```

Comment:
**suggestion:** If we are not using the `Message` in multiple places it is better to add it inside `src/components/Profile/`

Reply:
love it! moved it over to profile.